### PR TITLE
Added TensorFlow v2.0 implementation of FGSM Attack

### PR DESCRIPTION
**Pull request by rish-16**
### Description
I have written a TF v2.0 MNIST classifier with added support for the Keras functional API. I have incorporated the `art` modules like `KerasClassifier` and `FastGradientMethod`.

### System
I have tested it on macOS 10.14.14 Beta with different hyper-parameters. 

### Metrics
Original testing accuracy: 94.08
Adversarial testing accuracy: 45.16

### CODE REVIEW

**Comment by beat-buesser** [comment]
Hi @rish-16 Thank you very much for using ART! The example looks good, but I think it is very close to existing examples and notebooks, like `get_started_keras.py` or `art-for-tensorflow-v2-keras.ipynb` and therefore I'll close this PR. But please let me know if you would be interested to become a contributor to ART because could show you some interesting opportunities/project at various levels to develop code and new features for ART.

**Comment by rish-16** [comment]
Yes, please! I'd love to join in as a Contributor. I'm currently researching on Adversarial Examples at school and the ART community has been so helpful so far. Can I please know what are the next steps to take?

**Comment by beat-buesser** [comment]
Hi @rish-16 That great! Currently we have an open issue #51 to implement the low frequency adversarial perturbation strategy to complete our collection of attacks. Link to the paper is in the issue. I think it would be a great project to get started with a serious contribution. Would you be interested to work on this issue? If yes, I'll assign #51 to you and I can provide support to get you started.

**Comment by rish-16** [comment]
Sure! I looked through the paper just now. Let me see what I can do. I'll be writing it in TensorFlow/Keras. Worse case, it may need some low-level `tf.ops` but shouldn't be too much of a hassle.

Cheers!

**Comment by beat-buesser** [comment]
Great! I'll assign #51 to you and let's continue the discussion in issue #51.  Let me know anytime if you have questions.
All attacks in ART are framework (ML library) independent, which means they don't contain, for example, any TensorFlow code. All framework dependent code is implemented in the corresponding classifiers, e.g. TensorFlowV2Classifier, etc., which provide methods for loss gradients, etc.

**Comment by beat-buesser** [comment]
@rish-16 Could you please make a comment in #51 that you are interested in that issue? Otherwise I the system won't let me assign it to you.
